src/evmTest/scala/io/iohk/ethereum/vm/ThrowSpec.scala

```diff
@@ -13,7 +13,7 @@ class ThrowSpec extends FreeSpec with Matchers {
 
       val result = contract.justThrow().call()
 
-      result.error shouldBe Some(InvalidOpCode) // TODO: change to invalid jump destination
+      result.error shouldBe Some(InvalidOpCode(0xfd.toByte))
```

Comment:
Well, OK, that makes sense, maybe expect for the part that there is no revert instruction (0xfd) in the latest YP. 

But we've been over this? Don't you remember? We had a `StackOverflow` because jump destination were not implemented.

So what changed? The solidity version is the same as before, right?